### PR TITLE
feat: update deeplinks docs

### LINKED ISSUE #7751: [Task]: Update deeplinks documentation
### Task description

- [x] IOTA metric system is not allowed anymore, default SMR unit is glow, and default IOTA unit is micro
- [x] `If you wish to target the production version, simply omit this from the prefix` this doesnt make sense, you should just skip the stage for a production version
- [x] Remove `This deep link scheme is NOT compatible with Firefly V1, as` ....
- [x] Update send confirmation screenshot 
- [x] Explain that we have 2 deeplink schemes, `iota` for Firefly IOTA and `firefly` for Firefly Shimmer 
- [x] Review send confirmation & send form screenshots (use IOTA)
- [x] Review send confirmation & send form params to make sure they are accurate

### PR DESCRIPTION
**Pull request by evavirseda**
## Summary

> Closes #7751 

...

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: docs/specifications/deep-links.md
Comment (marc2332) [inline docs/specifications/deep-links.md:1]: I would call `SHIMMER` just `Shimmer

File: docs/specifications/deep-links.md [hunk 1]
```diff
@@ -40,15 +49,14 @@ The parameters are as follows:
 -   `operation` - an operation within a specific context (see below for more detail)
 -   `param` - query parameter(s) relevant for the specified operation
 
-If you wish to target the production version, simply omit this from the prefix:
+To target the production version, use the following deeplink prefix:
```
Comment (marc2332) [inline docs/specifications/deep-links.md:53]:
```suggestion
To target the production version simply don't specify any stages, for example with Shimmer:

```

**Review by marc2332 — CHANGES_REQUESTED** [review]
Thoughts?

**Review by cpl121 — APPROVED** [review]
LGTM

**Review by marc2332 — APPROVED** [review]

**Pull request merged**